client/src/pages/Home.jsx

```diff
@@ -1,6 +1,21 @@
+import "../styles/Home.css";
+import { NavLink } from "react-router-dom";
+// import {BsArrowRight} from "react-icons/bs"
+
 function Home() {
     return (
-        <p>Page d'accueil</p>
+        <>
+        {/* <p>Page d'accueil</p> */}
+        
+        <img className="gorille-art" src="../src/assets/images/gorille-art-bdx-2.jpg" alt="gorille-art" />
+        <h1 className="title-site">PixHunt</h1>
+        <p className="welcome-sentence">La chasse au <span className="street-art">Street Art</span> peut commencer !</p>
+        {/* <BsArrowRight className="bs-arrow-right"/> */}
+        <NavLink to="/mappage" ><button className="formButton" type="submit">
```

Comment:
It is useless to add a "button" to a "NavLink". You can add a className directly in the "NavLink": like that 
<NavLink to="/mappage" className="x" />...

Buttons are not made for this kind of method